ui/objects/semantic-ui/js/objects/editform/components/BasicInfo.jsx

```diff
@@ -28,6 +32,33 @@ export const BasicInfo = ({ activeIndex, handleActive, record }) => {
     { value: "metr", text: "metr" },
     { value: "mm", text: "mm" },
   ];
+
+  const { formConfig } = useFormConfig();
+
+  console.log(formConfig)
+
+  const filteredVocabularies = (item) => {
```

Comment:
`filteredVocabularies (item)` -> `vocabularyItemsInCategory(vocabularyName, category)` (clearer purpose of this function from its name & signature)

or even refactor & use `useVocabularyOptions` hook (see below) and then do `vocabularyOptionsInCategory(options, category)` and extract sorting logic to separate func `sortByText(options)`

Reply:
**UPDATE**: what happened to filtering of vocabularies by category? Is it happening no more? :)